sycl/include/sycl/handler.hpp

```diff
@@ -734,6 +774,16 @@ class __SYCL_EXPORT handler {
       ext::oneapi::experimental::detail::dynamic_parameter_base
           &DynamicParamBase,
       int ArgIndex);
+#endif
+
+  /// Registers a dynamic parameter with the handler for later association with
+  /// the node being created
+  /// @param DynamicParamImpl
+  /// @param ArgIndex
```

Comment:
These `@param` lines don't add any extra information (or the ones above), I'd either fill them in or just remove them.